docs/advanced-config/README.md

```diff
@@ -141,6 +141,25 @@ bleopt prompt_ps1_final='$(starship module character)'
 bleopt prompt_rps1_final='$(starship module time)'
 ```
 
+## TransientPrompt and TransientRightPrompt in Nushell
+
+Nushell allows enabling transient prompt by configuring enviromental variables
+inside of the `~/.config/nushell/env.nu` file. Transient prompt is
+automatically enabled when variable `TRANSIENT_PROMPT_COMMAND` is set. This
+variable controls what the left side of the input will be replaced with, valid
+values can be `""`, `"❯"` or a command prefixed with `^`. Similarly, variable
+`TRANSIENT_PROMPT_COMMAND_RIGHT` controls what the right side of the input will
+be replaced with. Example configuration for Starship prompt can be seen bellow.
```

Comment:
Please write this section more in line with the other transient prompt sections, and please add missing articles (the etc.) where appropriate.